src/applications/mhv/secure-messaging/containers/ThreadDetails.jsx

```diff
@@ -70,35 +64,16 @@ const ThreadDetails = props => {
             setIsLoaded(true);
           })
           .catch(() => {
-            navigateToFolderByFolderId(folder?.folderId || 0, history);
+            // navigateToFolderByFolderId(folder?.folderId || 0, history);
```

Comment:
Should this commented-out code be removed?

Reply:
I actually need to keep this, great catch